packages/contracts/src/contracts/current/protocol/Exchange/MixinExchangeCore.sol

```diff
@@ -177,8 +183,10 @@ contract MixinExchangeCore is
             orderHash: getOrderHash(orderAddresses, orderValues)
         });
 
+        require(order.makerTokenAmount > 0);
```

Comment:
Why this change?

Reply:
I used to do `assert(someThing && somethingElse && somethingOther)` a lot in a past project. Then one day we get a stack trace from a customer, and it says basically: "assertion failed on line ...". We wasted a lot of time trying to figure out which of the clauses failed and learned a hard lesson on never using `&&` in an assertion :)

If there is no performance difference, I prefer not using `&&` in assertions. Imho it makes it more readable and debuggable.